docs/ci-integration/ci-integration.md

```diff
@@ -0,0 +1,118 @@
+# Earthly in CI
+
+Continuous Integration systems are as varied as the companies that use them. Fortunately, Earthly is flexible enough to fit into most (and where we don't, let us know!). This document serves as a startping point to configuring Earthly in your CI environment.
+
+## Getting Started
+
+### Dependencies
+
+Earthly has two software dependencies: `docker` and `git`. 
+
+ `docker` is used to glean information about the containerization environment, and manage our `earthly-buildkitd` daemon. It is also used to do things like save images locally on your machine after they have been built by Earthly.
+
+Currently, the `earthly-buildkitd` daemon requires running in `--privileged` mode, which means that the `docker` daemon needs to be configured to allow this as well. Rootless configurations are currently unsupported.
+
+`git` is used to help fetch remote targets, and also provides metadata for Earthly during your build.
+
+Because `earthly` will not install these for you, please ensure they are present before proceeding. These tools are very common, so many environments will already have them installed. If they are not, here are some installation instructions that may help:
+
+#### Linux
+
+To install `git`, you can typically use your distributions package manager. [This page](https://git-scm.com/download/linux) has installation instructions for most distributions.
+
+To install `docker`, use the most recent versions [directly from Docker](https://docs.docker.com/engine/install/#server). The versions packaged for many distributions tend to fall behind.
+
+#### macOS
+
+To install `git`, the easiest way is to install the "XCode Command Line Tools". If you open up `Terminal`, and type:
+
+```go
+git --version
+```
+
+Then macOS will prompt you to install these tools. You can also use the `git` provided installer or Homebrew, if you prefer. [Details can be found here](https://git-scm.com/download/mac).
+
+To install `docker`, [download and install Docker CE](https://hub.docker.com/editions/community/docker-ce-desktop-mac). Be sure to grab the correct installer depending on your CPU architecture.
+
+#### Windows
+
+To install `git`, use the  [MSI installer](https://gitforwindows.org/). This will provide `git`, and a Bash shell; which may prove more natural for using Earthly. You may also use your package manager of choice.
+
+To install `docker`, [download and install Docker CE](https://hub.docker.com/editions/community/docker-ce-desktop-windows). Both the HyperV and WSL2 backends are supported, but the WSL2 one is very likely to be faster.
```

Comment:
I don't know if we should even include mac and windows CIs. Since Earthly only works with Linux containers, it would only make sense to use Earthly on a Linux-based CI.

Reply:
We ship a Mac and Windows build, and people might be using those in their CI. The fact that we only work with Linux containers shouldn't stop people from using the Windows or Mac runners IMO